### PR TITLE
feature: add info on Kotlin/Wasm

### PR DESCRIPTION
**Pull request by koshachy**

### CODE REVIEW

File: docs/topics/whatsnew-eap.md [hunk 1]
```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
...
+* Small size of the final binary
+
+Since 1.8.20-Beta preview release, you can use Kotlin/Wasm in your experimental projects.
+We provide Kotlin standard library (`stdlib`) and test library (`kotlin.test`) for Kotlin/Wasm out of the box.
 
 [Learn more about Kotlin/Wasm in the YouTube video](https://www.youtube.com/watch?v=-pqz9sKXatw).
 
+### How to enable the Kotlin/Wasm
+
+To create a new Kotlin/Wasm project with Project Wizard, install the EAP version of IntelliJ IDEA 2022.3 and select the
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:128]: Soon you also need on the registry flag kotlin.wasm.wizard ([KTIJ-24538](https://youtrack.jetbrains.com/issue/KTIJ-24538)), so it's probably are not the best way to promote now :(.

File: docs/topics/whatsnew-eap.md [hunk 2]
```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
...
+```kotlin
+plugins {
+   kotlin("multiplatform") version "1.8.20-Beta"
+}
+
+kotlin {
+   wasm {
+       binaries.executable()
+       browser {
+           commonWebpackConfig {
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:142]: it could be removed unless we write something there

File: docs/topics/whatsnew-eap.md [hunk 3]
```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
...
+   1. Go to `chrome://flags/#enable-webassembly-garbage-collection` in your browser.
+   2. Relaunch the browser application.
+
+* **Firefox**. For version 111 or newer:
+
+   1. Go to `about:config` in your browser.
+   2. Enable `javascript.options.wasm_function_references` and `javascript.options.wasm_gc` options.
+   3. Relaunch the browser application.
+
+* **Edge**.
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:176]: For version 109 or newer:

Comment (bashor) [reply docs/topics/whatsnew-eap.md:176]: Other option: is merging this with Chrome section, thought it still doesn't have flag in chrome://flags/

File: docs/topics/whatsnew-eap.md [hunk 4]
```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
...
-> The new backend is available in EAP of IntelliJ IDEA 2022.3 only.
->
-{type="note"}
+Kotlin/Wasm (Kotlin WebAssembly) goes [Experimental](components-stability.md#stability-levels-explained) in this preview release!
+Kotlin team finds [WebAssembly](https://webassembly.org/) is promising technology. We have a close look at it and 
+try to find better ways how you can use WebAssembly and get all the benefits of Kotlin.
+
+WebAssembly binary format is independent of the platform because it runs using its own virtual machine.
+Almost all the modern browsers support WebAssembly in an experimental mode, so you don't need to install additional tooling to 
+have an environment to run WebAssembly, therefore, Kotlin/Wasm.
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:113]: [The GC proposal](https://github.com/WebAssembly/gc) we rely on is still experimental and hidden under the flag.

Comment (bashor) [reply docs/topics/whatsnew-eap.md:113]: @koshachy important to fix!

File: docs/topics/whatsnew-eap.md [hunk 5]
```diff
@@ -104,14 +104,83 @@ We'd appreciate your feedback in any form:
...
+* Performance: fast speed of the compilation process
+* Faster integration with the host
+* Small size of the final binary
+
+Since 1.8.20-Beta preview release, you can use Kotlin/Wasm in your experimental projects.
+We provide Kotlin standard library (`stdlib`) and test library (`kotlin.test`) for Kotlin/Wasm out of the box.
 
 [Learn more about Kotlin/Wasm in the YouTube video](https://www.youtube.com/watch?v=-pqz9sKXatw).
 
+### How to enable the Kotlin/Wasm
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:126]: Is it possible to redirect https://kotl.in/wasm_help to here temporarily?

File: docs/topics/whatsnew-eap.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/whatsnew-eap.md]: Temporarily removed this part, added that the ide support will be in future releases

**Review by danil-pavlov — APPROVED** [review]

File: docs/topics/whatsnew-eap.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ _[Release date: 8 February 2023](eap.md#build-details)_
 The Kotlin 1.8.20-Beta release is out! Here are some highlights from this preview version of Kotlin:
 
 * [New Kotlin K2 compiler updates](#new-kotlin-k2-compiler-updates)
-* [New Kotlin/Wasm compiler backend](#new-kotlin-wasm-target)
+* [New Kotlin/Wasm target](#new-kotlin-wasm-target)
```
Comment (bashor) [inline docs/topics/whatsnew-eap.md:15]: experimental?

**Pull request merged**